pennylane/pulse/__init__.py

```diff
@@ -55,6 +55,8 @@
     ~pwc
     ~pwc_from_function
     ~rect
+    ~rydberg_interaction
+    ~rydberg_drive
```

Comment:
Should these just be added here under Convenience Functions, or do we want a separate section for Hardware Compatible Hamiltonians or similar?

Reply:
Also, this is probably more for a separate documentation PR once things are more settled closer to release, but we probably want a section in this qml.pulse module page about using hardware, right?